packages/web-components/fast-foundation/src/text-field/text-field.template.ts

```diff
@@ -2,6 +2,22 @@ import { html, ref, slotted } from "@microsoft/fast-element";
 import { endTemplate, startTemplate } from "../patterns";
 import { TextField } from "./text-field";
 
+/***
+ * A simple whitespace filter for the template
+ *
+ * @param value - The Node that is being inspected
+ * @param index - The index of the node within the array
+ * @param array - The Node array that is being filtered
+ *
+ * @public
+ */
+export function whitespaceFilter(value: Node, index: number, array: Node[]) {
```

Comment:
Kind of nitty but I don't think this should be exported from `text-field.template.ts` because there is nothing intrinsic to text-field about this filter. It could likely be exported from `fast-element` but it would be good to get @EisenbergEffect's opinion on that.

Reply:
I almost added it to the utilities section as I could see this getting used on `text-area` as well as it has a similar issue when whitespace is slotted, we just haven't noticed it yet because it does not have start and end slots.

Think i should move it to /utilities for now?

Reply:
I moved it to utils for now, as it should be able to be used elsewhere.